### ISSUE TITLE
Error Popup: When two admins approve a post in moderation by a difference of few seconds

### ISSUE THREAD

**Issue by gurjyot** [q2a/question2answer#392]
This happened today only, I opened the moderation page and saw a post there so I click the approve button. Then an error popup occurred. Then I saw the homepage of my website and the post was already approved by another admin. So why a sever error message popup during this scenario? Why not a message like "There is no post in moderation to be approved?" or "The post has already been approved or hidden."

The error message which pops up is this.

![error](https://cloud.githubusercontent.com/assets/6462850/13635286/bedc36b8-e620-11e5-9569-b2e1ad59508e.png)

**Comment by svivian** [comment]
Yes that's a fair point. It's not a simple fix though - the `qa_admin_single_click` only returns true or false, rather than a particular error message.
